ListingManager/ListingManager.cs

```diff
@@ -24,24 +24,30 @@ public static IEnumerable<string> GetAllExtraListings(string pathToStartFrom)
             }
         }
 
-        private static bool TryGetListing(string listingPath, out ListingInformation? listingData)
+        private static bool TryGetListing(string listingPath, out ListingInformation? listingData, bool onlyCSFiles = false)
```

Comment:
In general `out` parameters are typically listed last.

Reply:
Removed option alltogether